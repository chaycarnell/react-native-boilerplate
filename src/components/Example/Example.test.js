import React from 'react';
import { render } from '@testing-library/react-native';
import Example from './Example';

describe('Example component', () => {
  describe('given default props', () => {
    let renderedComponent;

    beforeEach(() => {
      renderedComponent = render(<Example />);
    });

    it('should render correctly', () => {
      const { toJSON } = renderedComponent;
      expect(toJSON()).toMatchSnapshot();
    });
  });
});
