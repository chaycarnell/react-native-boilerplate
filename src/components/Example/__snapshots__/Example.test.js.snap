// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example component given default props should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignSelf": "center",
        "backgroundColor": "green",
        "display": "flex",
        "height": 667,
        "justifyContent": "center",
        "opacity": 1,
        "width": 750,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 48,
          "fontWeight": "bold",
          "textAlign": "center",
        },
      ]
    }
  >
    Tap To Navigate
  </Text>
</View>
`;
